plugins/pulp_rpm/plugins/distributors/yum/metadata/updateinfo.py

```diff
@@ -63,7 +63,7 @@ def _get_repo_unit_nevra(self, erratum_unit):
         nevra_in_repo_and_pkglist = []
         for pkglist in erratum_unit.pkglist:
             for pkg in pkglist['packages']:
-                pkg_nevra = models.NEVRA(name=pkg['name'], epoch=pkg['epoch'],
+                pkg_nevra = models.NEVRA(name=pkg['name'], epoch=pkg['epoch'] or '0',
```

Comment:
@goosemania , filing a story to better validate the uploaded errata would be good. Thanks for verifying that the 'epoch' key is set during a sync.